src/main/java/com/pollapp/entity/Answer.java

```diff
@@ -0,0 +1,58 @@
+package com.pollapp.entity;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "answer")
+public class Answer {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    private String content;
+
+    @ManyToMany(fetch = FetchType.EAGER, mappedBy = "answers")
+    private List<UserData> data;
```

Comment:
Wouldn't it be easier to use the same name of the attribute as the entity it refers to? (e.g. List<UserData> userData)

Reply:
In this case i agree because userData also refers to plural